pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: gallery
 description: A resource to help developers evaluate and use Flutter.
 repository: https://github.com/flutter/gallery
-version: 2.4.0+020400
+version: 2.4.1
```

Comment:
Needs to be 2.4.1+020401

Reply:
yeah, this format was needed to make the play store happy

Reply:
Actually since this includes all those nice new motion demos, maybe 2.5.0 is more appropriate?